src/main/java/org/veupathdb/eda/binaryfiles/dumper/StudyDumper.java

```diff
@@ -153,15 +154,14 @@ private void writeMetaJsonFile(BinaryFilesManager.Metadata metadata, Entity enti
     }   
   }
   
-  private void writeDoneFile(Path directory) {
+  private void writeDoneFile(Path directory, long version) {
     try (FileWriter writer = 
         new FileWriter(_bfm.getDoneFile(directory, Operation.WRITE).toFile())) {
-      writer.write("");
-      writer.flush();
+      Map<String, Long> filesMeta = Map.of("dataVersion", version);
```

Comment:
I wonder if a more user-friendly timestamp would be helpful- to see exactly what the date was.  We can find out, of course.  Or is this value ingested and converted back to a DateTime for display somewhere?  If so, that would be sweet.